harness/determined/cli/project.py

```diff
@@ -0,0 +1,270 @@
+            render.format_time(e.endTime),
+            e.resourcePool,
+        ]
+        if args.all:
+            result.append(e.archived)
+        return result
+
+    headers = [
+        "ID",
+        "Owner",
+        "Name",
+        "Parent ID",
+        "State",
+        "Progress",
+        "Start Time",
+        "End Time",
+        "Resource Pool",
+    ]
+    if args.all:
+        headers.append("Archived")
+    values = [format_experiment(e) for e in experiments]
+    render.tabulate_or_csv(headers, values, False)
+
+
+def render_project(project: v1Project) -> None:
+    values = [
+        project.id,
+        project.name,
+        project.description,
+        project.numExperiments,
+        project.numActiveExperiments,
+    ]
+    PROJECT_HEADERS = ["ID", "Name", "Description", "# Experiments", "# Active Experiments"]
+    render.tabulate_or_csv(PROJECT_HEADERS, [values], False)
+
+
+def project_by_name(
+    sess: session.Session, workspace_name: str, project_name: str
+) -> Tuple[v1Workspace, v1Project]:
+    w = workspace_by_name(sess, workspace_name)
+    p = bindings.get_GetWorkspaceProjects(sess, id=w.id, name=project_name).projects
+    if len(p) == 0:
+        raise Exception(f'No project found on this workspace with name: "{project_name}"')
+    return (w, p[0])
+
+
+@authentication.required
+def list_project_experiments(args: Namespace) -> None:
+    sess = setup_session(args)
+    (w, p) = project_by_name(sess, args.workspace_name, args.project_name)
+    kwargs = {
+        "id": p.id,
+        "orderBy": bindings.v1OrderBy[f"ORDER_BY_{args.order_by.upper()}"],
+        "sortBy": bindings.v1GetProjectExperimentsRequestSortBy[f"SORT_BY_{args.sort_by.upper()}"],
+    }
+    if not args.all:
+        kwargs["users"] = [authentication.must_cli_auth().get_session_user()]
+        kwargs["archived"] = "false"
+    experiments = bindings.get_GetProjectExperiments(sess, **kwargs).experiments
+    if args.json:
+        print(json.dumps([e.to_json() for e in experiments], indent=2))
+    else:
+        render_experiments(args, experiments)
+
+
+@authentication.required
+def create_project(args: Namespace) -> None:
+    sess = setup_session(args)
+    w = workspace_by_name(sess, args.workspace_name)
+    content = bindings.v1PostProjectRequest(
+        name=args.name, description=args.description, workspaceId=w.id
+    )
+    p = bindings.post_PostProject(sess, body=content, workspaceId=w.id).project
+    if args.json:
+        print(json.dumps(p.to_json(), indent=2))
+    else:
+        render_project(p)
+
+
+@authentication.required
+def describe_project(args: Namespace) -> None:
+    sess = setup_session(args)
+    (w, p) = project_by_name(sess, args.workspace_name, args.project_name)
+    if args.json:
+        print(json.dumps(p.to_json(), indent=2))
+    else:
+        render_project(p)
+        print("\nAssociated Experiments")
+        users: List[str] = []
+        if args.all:
```

Comment:
super minor nit: with the logic here being the same as in list_project_experiments, would have preferred the code to read the same.

Reply:
✅ figured out an issue with kwargs here